### PR TITLE
Remove reference to disabling/enabling RID-500

### PR DESCRIPTION
**Pull request by tom-maher**
Propose that reference to disabling/enabling the builtin administrator account (RID-500) is removed throughout to retain consistency with the initial note ("This guide used to recommend disabling the account.").

### CODE REVIEW

**Comment by PRMerger12** [comment]
@tom-maher : Thanks for your contribution! The author(s) have been notified to review your proposed change. @iainfoulds

File: WindowsServerDocs/identity/ad-ds/plan/security-best-practices/Appendix-D--Securing-Built-In-Administrator-Accounts-in-Active-Directory.md [hunk 1]
```diff
@@ -61,11 +61,11 @@ For the built-in Administrator account in each domain in your forest, you should
...
 
 -   Configure Auditing of Administrator Accounts
 
-    When you have secured each domain's Administrator account and disabled it, you should configure auditing to monitor for changes to the account. If the account is enabled, its password is reset, or any other modifications are made to the account, alerts should be sent to the users or teams responsible for administration of Active Directory, in addition to incident response teams in your organization.
+    When you have secured each domain's Administrator account, you should configure auditing to monitor for usage of, or changes to, the account. If the account is used to logon, its password is reset, or any other modifications are made to the account, alerts should be sent to the users or teams responsible for administration of Active Directory, in addition to incident response teams in your organization.
```
Comment (illfated) [inline WindowsServerDocs/identity/ad-ds/plan/security-best-practices/Appendix-D--Securing-Built-In-Administrator-Accounts-in-Active-Directory.md:68]: 'to log on' is a compound verb, while 'logon' is the noun. Anyway, I think it should be phrased more like this:
```suggestion
    When you have secured each domain's Administrator account, you should configure auditing to monitor for usage of, or changes to, the account. If the account is logged in to, its password is reset, or any other modifications are made to the account, alerts should be sent to the users or teams responsible for administration of Active Directory, in addition to incident response teams in your organization.
```
Feel free to suggest a different phrasing, as long as it fits the general Microsoft Docs conventions.

**Comment by illfated** [comment]
Note: I am not familiar enough with this topic to know if the overall change is correct, so I leave that conclusion to the document author or manager.

**Comment by PRMerger16** [comment]
@tom-maher : Thanks for your contribution! The author(s) have been notified to review your proposed change. @iainfoulds

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
#sign-off

**Comment by PRMerger14** [comment]
@illfated: I'm sorry - only the author of this article, @iainfoulds, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by illfated** [comment]
Yes, I know. My sign-off comment above is based on a previous agreement with the team.

**Pull request merged**